store/src/java/com/zimbra/cs/datasource/CalDavDataImport.java

```diff
@@ -154,7 +154,10 @@ protected String getTargetUrl() {
             url.append("http://");
         break;
         }
-        url.append(ds.getHost()).append(":").append(ds.getPort());
+        url.append(ds.getHost());
```

Comment:
is there a port needed? may be this was earlier requirement for endpoint.

Reply:
Yes, this port is required here. I think it may be used by other services.